charts/ingress-nginx/templates/controller-poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (and (not .Values.controller.autoscaling.enabled) (gt (.Values.controller.replicaCount | int) 1)) }}
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
I think that it's a good idea to verify here the version of Kubernetes, so we wont break past users.

Example: https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/controller-ingressclass.yaml#L1

Reply:
Thanks for the suggestion. I added the validation and tested it in v1.21 and 1.20. Both versions work.